types/express-fileupload/index.d.ts

```diff
@@ -135,6 +135,11 @@ declare namespace fileUpload {
          * @default false
          */
         debug?: boolean | undefined;
+        /**
+         * Customizable logger to write debug messages to.
+         * @default console
+         */
+        logger?: { log: (msg: string) => void } | undefined;
```

Comment:
you mean something like (taking shape to match contract):
```ts
logger?: { log: Console["log"] } | undefined;
```

Reply:
Ok. Can do it this way.

Reply:
```suggestion
        logger?: { log: Console["log"] } | undefined;
```